src/scss/bootstrap.scss

```diff
@@ -4,4 +4,9 @@
 
 @import "bootstrap/bootstrap";
 
-// INSERT CUSTOM EXTENSIONS
\ No newline at end of file
+// INSERT CUSTOM EXTENSIONS
+
+// Prevent the focus on the dropdown toggle when closing dropdowns
+.dropdown-toggle:focus {
```

Comment:
We should use what Bootstrap uses instead of Lexicon styles, should use `@include tab-focus;` instead of `box-shadow`. This is Bootstrap only no Lexicon, also can you put the code between:
```
@import "bootstrap/bootstrap";

// Prevent the focus on the dropdown toggle when closing dropdowns

.dropdown-toggle:focus {
  @include tab-focus;
}

// INSERT CUSTOM EXTENSIONS
```

Reply:
Hi @pat270, thanks for your explanation. Requested changes already pushed.

Regards